packages/builder/src/components/common/bindings/DrawerBindableInput.svelte

```diff
@@ -13,7 +13,7 @@
   export let value: any = ""
   export let bindings: any[] = []
   export let title: string | undefined = undefined
-  export let placeholder: string | undefined = undefined
+  export let placeholder: string | false | undefined = undefined
```

Comment:
Why is false an option, can it not just be left undefined?

Reply:
I wish that was the case yea - we used to use an explicit `null` in some places to mean "don't show a placeholder option at all for selects", and someone changed that recently so that `false` for a placeholder now means that for selects. The problem is that there is a bunch of core logic that passes the placeholder into a variety of components, so we are passing `false` in to here at times too. So that's why it needs to support being a bool. Further down I just coerce this into `undefined` if it is false, so at least other components down the chain don't need to know about this weird typing.